jenkins/Jenkinsfile-blossom.premerge-databricks

```diff
@@ -91,7 +91,7 @@ pipeline {
                         // 'name' and 'value' only supprt literal string in the declarative Jenkins
                         // Refer to Jenkins issue https://issues.jenkins.io/browse/JENKINS-62127
                         name 'DB_RUNTIME'
-                        values '11.3', '12.2', '13.3'
+                        values '11.3', '12.2', '13.3', '14.3'
```

Comment:
Will need to drop support for 11.3 Databricks runtime @razajafri  chan you help to remove 11.3 here like
`values '11.3', '12.2', '13.3', '14.3'  --> values  '12.2', '13.3', '14.3'`

Reply:
Removing 11.3 should be a part of a different PR. 
This PR is not going to be merged as I am breaking it into smaller PRs